README.md

```diff
@@ -1,17 +1,29 @@
-## My Project
+## AWS C SDKUTILS
 
-TODO: Fill this README out!
+C99 library implementing AWS SDK specific utilitiies. Includes utilities for ARN
+parsing, reading AWS profiles, etc...
 
-Be sure to:
+## License
 
-* Change the title in this README
-* Edit your repository description on GitHub
+This library is licensed under the Apache 2.0 License.
 
-## Security
+## Usage
 
-See [CONTRIBUTING](CONTRIBUTING.md#security-issue-notifications) for more information.
+### Building
 
-## License
+CMake 3.0+ is required to build.
+
+`<install-path>` must be an absolute path in the following instructions.
+
+
+#### Building aws-c-s3 and Remaining Dependencies
```

Comment:
~aws-c-s3~ -> aws-c-sdkutils

Reply:
actually the `Remaining Dependencies` are not useful here as well

Reply:
ah, yeah copy-paste error. fixed in next commit